src/hooks/usePrevious.ts

```diff
@@ -0,0 +1,9 @@
+import { useEffect, useRef } from "react";
+
+export function usePrevious(value: any) {
```

Comment:
Funker det med `(value : string | number)` for å droppe å ha any? Også utvide vi med flere typer ved behov?

Reply:
Yes, skal få lagt på bedre typing!